_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
Actually we wouldn't need a new variable in config, because we already know the structure of the sections and chapters from [_contents.yml](https://github.com/doomspork/elixir-school/pull/1112/files#diff-71131372dfb994e3f094f9d9e2817101). So the index approach might make sense, but if we will finally decide to go this way, we should leave a comment to make clear what does the magic number mean.